portfolio/src/main/java/com/google/sps/servlets/LogoutServlet.java

```diff
@@ -0,0 +1,18 @@
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/logout-url")
+public class LogoutServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html");
+    String logoutUrl =
```

Comment:
The variable is "urlToRedirectToAfterLogin" but here it is logoutUrl? It is a little big confusing.

Reply:
Done. I agree, I missed that.

Reply:
Done. I agree, I missed this, thanks.